docssrc/src/kotlindsl.md

```diff
@@ -127,6 +127,36 @@ This works similar to how argument behaviour is modified in a CommandTree:
 
 </div>
 
+## Delegations
+
+The Kotlin DSL offers an additional way to access arguments: delegations. With delegations, there are two possible dependencies you can use:
```

Comment:
If my understanding is correct, isn't the technical term for what we've implemented here "delegated properties"? According to the [Kotlin documentation for Delegations](https://kotlinlang.org/docs/delegation.html), a delegation is a design pattern for inheritance, whereas the [Kotlin documentation for Delegated properties](https://kotlinlang.org/docs/delegated-properties.html) looks more akin to what we've implemented.

That said, a link to the Kotlin documentation for this pattern would be appreciated. I presume this sort of thing is much more common in the Kotlin developer scene, but it would be a good idea to point new Kotlin developers to the corresponding documentation.